q2_demux/plugin_setup.py

```diff
@@ -135,7 +138,9 @@
               'average positional qualities across all of the sequences '
               'selected. If input sequences are paired end, plots will be '
               'generated for both forward and reverse reads for the same `n` '
-              'sequences.')
+              'sequences.'),
+        'delta': ('The acceptable delta cutoff for disparities in sequence '
+                  'length')
```

Comment:
I think this could probably be improved, `delta` on it's own isn't super informative. Perhaps `acceptable_length_difference`?